ecosystem-modules/ren/renvm-bridge/src/lib.rs

```diff
@@ -107,13 +121,20 @@ decl_module! {
 		fn mint(
 			origin,
 			who: T::AccountId,
-			_p_hash: [u8; 32],
+			p_hash: [u8; 32],
 			#[compact] amount: Balance,
-			_n_hash: [u8; 32],
+			n_hash: [u8; 32],
 			sig: EcdsaSignature,
 		) {
 			ensure_none(origin)?;
-			Self::do_mint(who, amount, sig)?;
+			Self::do_mint(&who, amount, &sig)?;
+
+			let len = (32 + p_hash.len() + n_hash.len() + sig.0.len()) as u32;
```

Comment:
this is not accurate as it doesn't include everything in a transaction.
should just make build a dummy mint transaction, measure the length, and hardcode the value here